app/services/albums_service.rb

```diff
@@ -0,0 +1,25 @@
+class AlbumsService
+  include HTTParty
```

Comment:
If HTTParty is included in the gemfile, I don't think you need to use the include